app/services/transaction_search/searcher.rb

```diff
@@ -31,10 +31,16 @@ def self.search(order_details, params)
       new.search(order_details, params)
     end
 
+    # Shorthand method for searchers used on the billing page
+    def self.billing_search(order_details, params, include_facilities = false)
```

Comment:
I might consider making `include_facilities` a keyword argument. After about 3 arguments, I like to start naming them, especially optional ones.

Reply:
👍